sessionctx/variable/sysvar.go

```diff
@@ -581,3 +581,27 @@ type GlobalSysVarAccessor interface {
 	// SetGlobalSysVar sets the global system variable name to value.
 	SetGlobalSysVar(ctx context.Context, name string, value string) error
```

Comment:
This is implemented in session. It has many functions. So get/set is too simple.